sklearn/mixture/tests/test_gmm.py

```diff
@@ -108,6 +108,16 @@ def test_lmvnpdf_full():
     assert_array_almost_equal(lpr, reference)
 
 
+def test_lvmpdf_full_cv_non_positive_definite():
+    n_features, n_components, n_samples = 2, 1, 10
+    X = rng.randint(10) * rng.rand(n_samples, n_features)
+    mu = np.mean(X, 0)
+    cv = np.array([[-1, 0], [0, 1]])
+    assert_raises(ValueError,
```

Comment:
OK. I found in test_gmm.py still use testing functions of Numpy instead of sklearn own testing functions like in other modules. Should I use sklearn own testing functions?